index.js

```diff
@@ -62,13 +62,21 @@ function pick(opts) {
     );
   }
 
+  if ('mode' in opts && !['import', 'open'].includes(opts.mode)) {
+    throw new TypeError('Invalid mode option: ' + opts.mode);
+  }
+
   if ('copyTo' in opts && !['cachesDirectory', 'documentDirectory'].includes(opts.copyTo)) {
     throw new TypeError('Invalid copyTo option: ' + opts.copyTo);
   }
 
   return RNDocumentPicker.pick(opts);
 }
 
+function releaseSecureAccess(uri) {
```

Comment:
this fn seems a little redundant, perhaps it'd make sense to call it inline in `static releaseSecureAccess`? Also, question, what is the return value here? `releaseSecureAccess` is defined as returning `void`

Reply:
You're right it's a mistake, I will fix this.